omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -39,7 +39,9 @@ public void started() {
 
 	@Override
 	public void contextRefreshed() {
-		if (!Paths.get(SpaDirectoryResolver.getSpaDirectory(), "index.html").toFile().exists()) {
+		String spaDirectory = SpaDirectoryResolver.getSpaDirectory();
+		if (spaDirectory == null) return;
```

Comment:
Coding convention is to **always** use curly braces.

```suggestion
		if (spaDirectory == null) {
		    return;
		}
```